Update related documents with new ESP32-WROOM-32E & ESP32-WROOM-32UE datasheet (IDFGH-10262)

Luftalian opened the pull request:
This pull request updates the related documents to include the new ESP32-WROOM-32E and ESP32-WROOM-32UE datasheet.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/espressif/esp-idf?pullRequest=11524) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/espressif/esp-idf?pullRequest=11524) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/espressif/esp-idf?pullRequest=11524) it.</sub>

Luftalian commented:
- Updated the list of related documents to include the ESP32-WROOM-32E & ESP32-WROOM-32UE datasheet.
- Added a link to the ESP32-WROOM-32E & ESP32-WROOM-32UE datasheet for easy access.

Please review and merge this pull request!

ESP-Marius commented:
sha=b40158c9e55bd6abb0544c711b729d37631cb386

Alvin1Zhang commented:
Thanks for contribution again, changes have been merged with https://github.com/espressif/esp-idf/commit/87757f0a660ca26e0223bcd81ec330f598398229.
